server/test/instant/db/transaction_test.clj

```diff
@@ -26,6 +26,14 @@
   (:import
    (java.util UUID)))
 
+(test/use-fixtures :each
```

Comment:
Can we add a with-redefs that captures the outputs of `transact!` from each run and then fail the test if there are any differences? Might have to write something to replace uuids and tx-id, but everything else should be the same?

The data gets sent to the client when they run queries from the dashboard's sandbox, so the output format is kind of important.